server/backend/api/app/schemas.py

```diff
@@ -293,15 +311,19 @@ class Requester(BaseModel):
     """
     Class holding information for user who access FAM system after authenticated.
     """
+
     # cognito_user_id => Cognito OIDC access token maps this to: username (ID token => "custom:idp_name" )
     cognito_user_id: Union[str, None] = None
     user_name: Annotated[str, StringConstraints(max_length=20)]
     # "B"(BCeID) or "I"(IDIR). It is the IDP provider.
     user_type_code: Union[famConstants.UserType, None] = None
-    access_roles: Union[List[Annotated[str, StringConstraints(max_length=50)]], None] = None
+    access_roles: Union[
+        List[Annotated[str, StringConstraints(max_length=50)]], None
+    ] = None
+    user_guid: Optional[Annotated[str, StringConstraints(max_length=32)]] = None
```

Comment:
Added user_guid for requester, we need requester's guid to make api call for searching bceid